File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/chart/GetChartArea.java
package com.aspose.cells.cloud.examples.chart;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Path;

public class GetChartArea {

    public static void main(String... args) throws IOException {
        // ExStart: get-chart-area
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            String input = "Sample1.xlsx";
            Path inputFile = Utils.getPath(GetChartArea.class, input);
            String sheet = "Sheet1";
            int chartIndex = 0;

            storageApi.PutCreate(input, null, Utils.STORAGE, inputFile.toFile());

            com.aspose.cells.model.ChartAreaResponse apiResponse = cellsApi.GetChartArea(input, sheet, chartIndex,
                    Utils.STORAGE, Utils.FOLDER);

            System.out.println("Area X: " + apiResponse.getChartArea().getX());
            System.out.println("Area Y: " + apiResponse.getChartArea().getY());
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: get-chart-area
    }
}

File: Examples/DotNet/CSharp/Charts/GetChartArea.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Charts
{
    class GetChartArea
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "Sample_Test_Book.xls";
            String sheetName = "Sheet5";
            int chartIndex = 0;
            String storage = "";
            String folder = "";
            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to get chart area
                ChartAreaResponse apiResponse = cellsApi.GetChartArea(fileName, sheetName, chartIndex, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    ChartArea chartArea = apiResponse.ChartArea;
                    Console.WriteLine("ChatArea X :: " + chartArea.X);
                    Console.WriteLine("ChatArea Y :: " + chartArea.Y);
                    Console.WriteLine("ChatArea Area :: " + chartArea.Area);
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
